compiler/src/iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -2829,10 +2823,8 @@ class CallOpConversion : public EmitCConversionPattern<OpTy> {
 
     Value operandRef = this->getModuleAnalysis().lookupRef(operand);
 
-    auto refOp = builder.create<emitc::VariableOp>(
-        /*location=*/loc,
-        /*resultType=*/emitc::OpaqueType::get(ctx, "iree_vm_ref_t"),
-        /*value=*/emitc::OpaqueAttr::get(ctx, ""));
+    auto refOp = allocateVariable(builder, loc,
+                                  emitc::OpaqueType::get(ctx, "iree_vm_ref_t"));
 
     Value refPtr = emitc_builders::addressOf(builder, loc, refOp.getResult());
```

Comment:
```suggestion
    Value ref = allocateVariable(builder, loc,
                                  emitc::OpaqueType::get(ctx, "iree_vm_ref_t"));

    Value refPtr = emitc_builders::addressOf(builder, loc, ref);
```

Reply:
Thanks, good fetch. Couldn't apply via GH as this clang-format didn't liked the suggested changed ;)